ts/features/wallet/component/FeaturedCardCarousel.tsx

```diff
@@ -0,0 +1,113 @@
+import { reverse } from "fp-ts/lib/Array";
+import * as pot from "italia-ts-commons/lib/pot";
+import { View } from "native-base";
+import * as React from "react";
+import { ScrollView } from "react-native";
+import { connect } from "react-redux";
+import { fromNullable } from "fp-ts/lib/Option";
+import { nullType } from "io-ts";
+import { getValue } from "../../bonus/bpd/model/RemoteValue";
+import {
+  ID_BONUS_VACANZE_TYPE,
+  ID_BPD_TYPE
+} from "../../bonus/bonusVacanze/utils/bonus";
+import { getLocalePrimaryWithFallback } from "../../../utils/locale";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import { H3 } from "../../../components/core/typography/H3";
+import I18n from "../../../i18n";
+import { GlobalState } from "../../../store/reducers/types";
+import { Dispatch } from "../../../store/actions/types";
+import { bpdEnabled } from "../../../config";
+import { bpdEnabledSelector } from "../../bonus/bpd/store/reducers/details/activation";
+import { bpdOnboardingStart } from "../../bonus/bpd/store/actions/onboarding";
+import { BonusAvailable } from "../../../../definitions/content/BonusAvailable";
+import { navigateToBonusRequestInformation } from "../../bonus/bonusVacanze/navigation/action";
+import { navigationHistoryPop } from "../../../store/actions/navigationHistory";
+import { availableBonusTypesSelector } from "../../bonus/bonusVacanze/store/reducers/availableBonusesTypes";
+import FeaturedCard from "./FeaturedCard";
+
+type Props = {
+  bvActive: boolean;
+} & ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const FeaturedCardCarousel: React.FunctionComponent<Props> = (props: Props) => {
+  const handlersMap: Map<number, (bonus: BonusAvailable) => void> = new Map<
+    number,
+    (bonus: BonusAvailable) => void
+  >([[ID_BONUS_VACANZE_TYPE, bonus => props.navigateToBonusRequest(bonus)]]);
+
+  if (bpdEnabled) {
+    handlersMap.set(ID_BPD_TYPE, _ => props.startBpdOnboarding());
+  }
+
+  const isOneBonusActive = !props.bvActive || !getValue(props.bpdActiveBonus);
```

Comment:
```suggestion
  const anyBonusNotActive = !props.bvActive || !getValue(props.bpdActiveBonus);
```
This is could be a better naming if I hit the point